aries-site/src/themes/aries.js

```diff
@@ -41,16 +29,25 @@ export const aries = deepMerge(hpe, {
     //   border: { side: 'bottom', size: 'xsmall' },
     // },
     header: {
-      pad: { horizontal: 'small', vertical: 'xsmall' },
+      // background: undefined,
+      border: { side: 'bottom' },
       color: 'text-strong',
       font: {
         weight: 'bold',
+        // size: undefined,
       },
+      // gap: 'small',
```

Comment:
Looking back at designs, I think we'll actually want this to be `gap="none"` so that once we add in any resizing, there won't be any space between the button and the resizer

Reply:
Added.